recipes/ta-lib/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "TA-Lib" %}
+{% set version = "0.4.16" %}
+{% set sha256 = "555f5d9e6720ef935669c1f404a7179091ae35869c6435957709a7059b9fd1d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
Add the conda-forge compiler packages (docs (here)[https://conda-forge.org/docs/meta.html#compilers]) please.
You no longer need the toolchain package